src/js/services/conversation-service.js

```diff
@@ -89,7 +89,7 @@ export function sendMessage(text) {
 
         const {user} = store.getState();
 
-        return core().conversations.sendMessage(getUserId(), message).then((response) => {
+        return core().appUsers.sendMessage(getUserId(), message).then((response) => {
             if (!user.conversationStarted) {
```

Comment:
This is done through the `SET_MESSAGES` action. This gets dispatched when connecting to Faye.